### PR TITLE
Publish 2024 OpenSearch release schedule

### PR DESCRIPTION
**Pull request by bbarani**
### Description
Publish 2024 OpenSearch release schedule
 
### Issues Resolved
- https://github.com/opensearch-project/.github/issues/186

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: releases.md [hunk 1]
```diff
@@ -21,12 +21,24 @@ breadcrumbs:
...
-| 2.12.0        | February 6th, 2024    | February 20th, 2024  | [Prudhvi Godithi](https://github.com/prudhvigodithi) | [4115](https://github.com/opensearch-project/opensearch-build/issues/4115) |
-| Being finalized   | TBD    | TBD | TBD | [Proposal - 186](https://github.com/opensearch-project/.github/issues/186) |
+  
+
+| Release Number| First RC Generated (release window opens) | Latest Possible Release Date (release window closes) | Release Manager                                     | Tracking Issue |
+|:--------------|:------------------------------------------|:---------------|:----------------------------------------------------|:---------------|
+| 2.12.0        | February 6th, 2024                        | February 20th, 2024  | [Prudhvi Godithi](https://github.com/prudhvigodithi) | [4115](https://github.com/opensearch-project/opensearch-build/issues/4115) |
+|1.3.15 | Feb 27 2024                               | March 05 2024|                                                     |[4294](https://github.com/opensearch-project/opensearch-build/issues/4294)
+2.13.0 | March 19th, 2024                          | April 2nd, 2024
```
Comment (peternied) [inline releases.md:30]: In the markdown preview this table isn't rendering correctly, I think its due to missing `|` characters.

Comment (dtaivpp) [reply releases.md:30]: Whats your environment setup like? When I pull this it's rendering alright

Comment (peternied) [reply releases.md:30]: Maybe the website markdown is processed differently that GitHub's preview, see how it looks here:
https://github.com/opensearch-project/project-website/pull/2588/files?short_path=1d2b91d#diff-1d2b91d957f8612017f196e6cc2e83c33bcfe3fd348dc3619e9e9672e797d46f

Comment (dtaivpp) [reply releases.md:30]: Ah that is it indeed it seems. Going to push through then because the Jekyll rendered version seems okay.

**Review by dtaivpp — APPROVED** [review]
Looks good!

**Pull request merged**